[docs] [PLAT-7426] added http/https proxy setup (#17600)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/default.md

![Application Config](/images/replicated/application-config-tls.png)

The recommended TLS version is 1.2.

## Set up HTTP/HTTPS proxy

> You can setup HTTP and HTTPS proxy via **Application config**, and select **Enable Proxy** as per the following illustration:

Reviewer: ```suggestion
You can set up HTTP and HTTPS proxies via **Application config**, and select **Enable Proxy** as per the following illustration:
```

![Enable Proxy](/images/replicated/enable-proxy.png)

The following information describes details about the **Enable Proxy** setting and some considerations.

> - If your proxy is using default ports for protocol, then use the defaults for HTTP and HTTPS, 80 and 443 respectively, instead of 8080 and 8443 as shown in the above illustration.

Reviewer: ```suggestion
- If your proxy is using the default ports for each protocol, then set the ports for the HTTP and HTTPS proxies to the default, 80 and 443 respectively, instead of 8080 and 8443 as shown in the preceding illustration.
```

> - If you have only one proxy setup (HTTP or HTTPS), then set the same values for both. This configuration ends up setting OS environment variables or java system properties.

Reviewer: ```suggestion
- If you have only one proxy set up (HTTP or HTTPS), then set the same values for both. This configuration sets operating system environment variables and Java system properties.
```

- The Java fields can accept values as Java system properties, including the use of pipe ("|") as a field separator. Refer to [Java Networking and Proxies](https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html) for more details about the properties.

> - There is no authoritative source for the format of environment variables; for setting proxy where two environment variables get exported with lowercase and uppercase formats for the fields. For example, if you enter "http://my.Proxy.host:8080" for **HTTP Proxy setting**, then two environment variables gets exported as follows:

Reviewer: did something get lost here

Author: No I just included an example for clarity : https://github.com/yugabyte/yugabyte-db/pull/17600#issuecomment-1572692176
